lib/slack-ruby-bot/app.rb

```diff
@@ -3,26 +3,36 @@
 module SlackRubyBot
   class App < Server
     def initialize(options = {})
-      SlackRubyBot.configure do |config|
-        config.token = ENV['SLACK_API_TOKEN'] || raise("Missing ENV['SLACK_API_TOKEN'].")
-        config.aliases = ENV['SLACK_RUBY_BOT_ALIASES'].split(' ') if ENV['SLACK_RUBY_BOT_ALIASES']
-      end
-      Slack.configure do |config|
-        config.token = SlackRubyBot.config.token
-      end
-      super
+      server_options = { token: options[:token] }
+      super(server_options)
```

Comment:
So this is `super(options)`, which is basically the same thing. Remove the constructor altogether, the `super`'s constructor works out of the box.